src/components/Blocking/ItemListLarge.js

```diff
@@ -53,13 +53,13 @@ const styles = StyleSheet.create({
   label: {
     marginLeft: 15,
     fontFamily: fonts.inter[500],
-    color: COLORS.black,
+    color: COLORS.white2,
     fontSize: 14
   },
   desc: {
     fontFamily: fonts.inter[400],
     fontSize: 12,
-    color: COLORS.blackgrey,
+    color: COLORS.gray500,
```

Comment:
<img src="https://avatars.githubusercontent.com/in/347564?s=41" alt="Image description" width="20" height="20">   CodeRabbit

The color of the label and desc text has been changed. The label's color has been changed from black to white2, and the desc's color has been changed from blackgrey to gray500. These changes seem to be in line with the PR's purpose of changing colors. However, please ensure that these color changes do not affect the visibility and readability of the text, especially considering different screen brightness settings and potential color blindness issues among users.
```diff
- color: COLORS.black,
+ color: COLORS.white2,
- color: COLORS.blackgrey,
+ color: COLORS.gray500,
```


<!-- This is an auto-generated comment by OSS CodeRabbit -->